iothub/device/src/Http1TransportSettings.cs

```diff
@@ -40,6 +40,9 @@ public TransportType GetTransportType()
         /// <summary>
         /// The time to wait for a receive operation. The default value is 1 minute.
         /// </summary>
+        /// <remarks>
+        /// This property is currently unused.
+        /// </remarks>
         public TimeSpan DefaultReceiveTimeout => s_defaultOperationTimeout;
```

Comment:
I could not find usages of this. Can someone else also confirm?

Reply:
This is quite possibly code left over from previous design, when we used to only support amqp and thus did not have the other transport protocol specific settings.